GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release and Long Term Support (LTS) teams. It
+manages the release and long term support shedule and policies
+for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
```

Comment:
Again, my understanding (but I can be wrong on this) is that it is the _same_ as most WGs. Streams has final authority about what goes into `readable_stream`, for example. You've never done anything objectionable, so there has never been conflict, and no one expects there to be, but its the streams WG who decides on what happens with streams. Maybe the WG charters need to change to saying that anything they decide can be overruled by a vote of CTC without dissolving the WG?